src/patterns/gen-ai/aws-model-deployment-sagemaker/jumpstart-sagemaker-endpoint.ts

```diff
@@ -195,7 +199,8 @@ export class JumpStartSageMakerEndpoint extends SageMakerEndpointBase {
     vpcConfig: sagemaker.CfnModel.VpcConfigProperty | undefined,
   ) {
     const key = this.spec.prepackedArtifactKey ?? this.spec.artifactKey;
-    const bucket = JumpStartConstants.JUMPSTART_LAUNCHED_REGIONS[this.region]?.contentBucket;
+    const bucket = this.spec.gatedBucket ? JumpStartConstants.JUMPSTART_LAUNCHED_REGIONS[this.region]?.gatedContentBucket :
```

Comment:
Please help me understand if adding the `gatedBucket` boolean will first be pulled.

Reply:
apologise, I don't understand your statement.
if gatedBucket is true, we will use `gatedContentBucket` otherwise use `contentBucket`.
At least that's my observation during deploying llama, please feel free to correct me.

Reply:
> apologise, I don't understand your statement. if gatedBucket is true, we will use `gatedContentBucket` otherwise use `contentBucket`. At least that's my observation during deploying llama, please feel free to correct me.

I miss interpreted that each model spec determines whether or not the `gatedBucket` exists or not.